
A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.2.5 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/Users/shubhi/Desktop/MT_Exercises/joeynmt-hotfixed/joeynmt/__main__.py", line 3, in <module>
    from joeynmt.prediction import test, translate
  File "/Users/shubhi/Desktop/MT_Exercises/joeynmt-hotfixed/joeynmt/prediction.py", line 19, in <module>
    from joeynmt.data import load_data
  File "/Users/shubhi/Desktop/MT_Exercises/joeynmt-hotfixed/joeynmt/data.py", line 9, in <module>
    from joeynmt.datasets import BaseDataset, build_dataset
  File "/Users/shubhi/Desktop/MT_Exercises/joeynmt-hotfixed/joeynmt/datasets.py", line 27, in <module>
    CPU_DEVICE = torch.device("cpu")
/Users/shubhi/Desktop/MT_Exercises/joeynmt-hotfixed/joeynmt/datasets.py:27: UserWarning: Failed to initialize NumPy: _ARRAY_API not found (Triggered internally at /Users/runner/work/pytorch/pytorch/pytorch/torch/csrc/utils/tensor_numpy.cpp:77.)
  CPU_DEVICE = torch.device("cpu")
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/Users/shubhi/Desktop/MT_Exercises/joeynmt-hotfixed/joeynmt/__main__.py", line 61, in <module>
    main()
  File "/Users/shubhi/Desktop/MT_Exercises/joeynmt-hotfixed/joeynmt/__main__.py", line 41, in main
    train(cfg_file=args.config_path, skip_test=args.skip_test)
  File "/Users/shubhi/Desktop/MT_Exercises/joeynmt-hotfixed/joeynmt/training.py", line 794, in train
    model_dir = make_model_dir(
  File "/Users/shubhi/Desktop/MT_Exercises/joeynmt-hotfixed/joeynmt/helpers.py", line 48, in make_model_dir
    raise FileExistsError(f"Model directory {model_dir} exists "
FileExistsError: Model directory /Users/shubhi/Desktop/MT_Exercises/mt-exercise-03/models/deen_transformer_pre exists and overwriting is disabled.
